bindings.rs

```diff
@@ -0,0 +1,6323 @@
+/* automatically generated by rust-bindgen 0.59.2 */
```

Comment:
Why is this file committed? It should be generated in OUTDIR/bindings.rs by the build script and never committed, as it is an artifact of the build process.